cpp/benchmarks/common/generate_input.cu

```diff
@@ -655,6 +717,23 @@ std::unique_ptr<cudf::table> create_sequence_table(std::vector<cudf::type_id> co
   return std::make_unique<cudf::table>(std::move(columns));
 }
 
+std::pair<rmm::device_buffer, cudf::size_type> create_random_null_mask(cudf::size_type size,
+                                                                       float null_probability,
+                                                                       unsigned seed)
+{
+  if (null_probability < 0.0f) {
+    return {rmm::device_buffer{}, 0};
+  } else if (null_probability == 0.0f) {
+    return {cudf::create_null_mask(size, cudf::mask_state::ALL_NULL), size};
+  } else if (null_probability >= 1.0f) {
+    return {cudf::create_null_mask(size, cudf::mask_state::ALL_VALID), 0};
```

Comment:
These seem backwards to me. Shouldn't `null_probability >= 1.0f` result in `ALL_NULL`?

Reply:
Good catch. updated.